cmake/modules/FindUSDMonolithic.cmake

```diff
@@ -75,23 +75,23 @@ if(USDMonolithic_FOUND)
         )
     endif()
 
-    add_library(usd_ms SHARED IMPORTED)
-    set_property(TARGET usd_ms APPEND PROPERTY IMPORTED_CONFIGURATIONS RELEASE)
+    add_library(usd_usd_ms SHARED IMPORTED)
+    set_property(TARGET usd_usd_ms APPEND PROPERTY IMPORTED_CONFIGURATIONS RELEASE)
```

Comment:
Why do you need to rename the target? There are no reasons for that. If we really need to rename it, then something like "usd_monolithic" would be more expressive